README.md

```diff
@@ -14,19 +14,29 @@ Basic Configuration in the `mkdocs.yml` file:
 
 ```yaml
 plugins:
-  - mkdocs_exclude_unused_files:
+  - exclude-unused-files:
```

Comment:
Hmmm, while I like dropping the mkdocs prefix, and also don't have a hard aversion against using hyphens instead of underscores, I'm unsure about this, since it's quite a breaking change.

I know this plugin isn't wildly popular, but there are nonethless some downloads[1] and I don't want to break anyones configuration with this update. 

What do you think regarding that?


[1]: https://pypistats.org/packages/mkdocs-exclude-unused-files

Reply:
Oh nvm, totaly read over the part where you already said that this won't break the old entrypoints, my bad!